interface/weno/logsync.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ *  @package OpenEMR
+ *  @link    http://www.open-emr.org
+ *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c) 2020 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once("../globals.php");
+
+use OpenEMR\Common\Logging\EventAuditLogger;
+use OpenEMR\Rx\Weno\Container;
+
+//TODO convert this entire file to a class
+$container = new Container();
+
+$wenoProperties = $container->getTransmitproperties();
+$provider_info = $wenoProperties->getProviderEmail();
+$logsync = $container->getLogproperties();
+
+/**
+ * checks to see if the file exist and if it does was it put there today?
+ * The idea behind this is to automate the log information download and import into the database.
+ * This should only execute once a day no matter how many times it is called.
+ * The idea was to include in the index file to be executed when the prescription called.
+ */
+$today = date("F d Y");
+$filedate = $logsync->doesLogFileExist();
+//die if the dates match or the file does not exist
+if ($today !== $filedate) {
+    $logurlparam = $logsync->logEcps();
+    $syncLogs = "https://test.wenoexchange.com/en/EPCS/DownloadNewRxSyncDataVal?useremail=";
+    if ($logurlparam == 'error') {
+        echo xlt("Cipher failure check encryption key");
+        exit;
+    }
+
+    $urlOut = $syncLogs . $provider_info['email'] . "&data=" . urlencode($logurlparam);
```

Comment:
place a comment above this line warning to not add urlencode() to the $provider_info['email'] variable